datadog_checks_dev/datadog_checks/dev/tooling/templates/check/{check_name}/README.md

```diff
@@ -12,15 +12,13 @@ This check monitors [{check_name_cap}][1].
 
 ### Configuration
 
-1. Edit the `{check_name}.d/conf.yaml` file, in the `conf.d/` folder at the root of your
-   Agent's configuration directory to start collecting your {check_name} performance data.
-   See the [sample {check_name}.d/conf.yaml][2] for all available configuration options.
+1. Edit the `{check_name}.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your {check_name} performance data. See the [sample {check_name}.d/conf.yaml][2] for all available configuration options.
 
-2. [Restart the Agent][3]
+2. [Restart the Agent][3].
 
 ### Validation
 
-[Run the Agent's `status` subcommand][4] and look for `{check_name}` under the Checks section.
+[Run the Agent's status subcommand][4] and look for `{check_name}` under the Checks section.
```

Comment:
Was there a reason to remove the backticks on status?

Reply:
Looks bad on the website with it being a link